index.htm

```diff
@@ -4,9 +4,29 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Blue Recycling</title>
 </head>
 <body>
+    <div id="navBar">
```

Comment:
Should we use the same case type throughout the project? You use camel case here and kebab on line 19.

Reply:
absolutely

Reply:
Great job!